main.js

```diff
@@ -59,7 +76,57 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function addBreakpointListeners() {
+	try {
+		// Create a map containing all breakpoints exposed via html:before
+		const breakpoints = new Map(Object.entries(getGridBreakpoints().layouts));
+		const decr1 = val => `${Number(val.replace('px', '') - 1)}px`;
```

Comment:
@JakeChampion Which, the `decr1` function? It's because (if I understand correctly) each breakpoint is defined as `(min-width: $current.startingValue) and (max-width: $next.startingValue - 1)`.